### DISCUSSION TITLE
How to display thumbnail in a collection?

### DISCUSSION THREAD

**Discussion by orasik** [payloadcms/payload#3407] [General]
Hi,
I have a collection with an upload field. The upload works fine; I can see the image when editing the record.

However, I would like to display the thumbnail in the collection list. Is there a config to do this without writing a custom component?

Thanks!

Ex:
<img width="1540" alt="ss-2023-09-29-09 25 17" src="https://github.com/payloadcms/payload/assets/9113844/97de97bb-8027-456c-bda5-db23fabe37b7">

**Comment by revnelson** [comment]
This would be great if any relationship field that is related to an upload collection defaults to using a thumbnail in the collection list view, with an option to use any other field on that related item instead.

**Comment by rklos** [comment]
I'm also struggling with this problem

edit: I've made a PR with this feature: https://github.com/payloadcms/payload/pull/5015
